parts/linux/cloud-init/artifacts/cse_main.sh

```diff
@@ -336,6 +336,28 @@ else
             systemctl restart --no-block apt-daily.service
         fi
         aptmarkWALinuxAgent unhold &
+    elif [[ $OS == $MARINER_OS_NAME ]]; then
```

Comment:
if isMariner && !isKata removes the need for conditional below

Reply:
I was going to leave the echo that explains why we aren't turning on the updates in this specific case, but if we don't want that happy to tidy that up.

Reply:
ah no the comments are fine. just the go templates was all